site/go.mod

```diff
@@ -0,0 +1,10 @@
+module github.com/bep/docuapi
```

Comment:
Is this correct?

Why do we need a sub module? And if we do, isn’t there a `go.sum` file?